src/CFamily.UnitTests/CMake/CompilationDatabaseLocatorTests.cs

```diff
@@ -313,11 +305,17 @@ private void SetupCMakeSettingsFileExists(Mock<IFileSystem> fileSystem, string c
             return new CompilationDatabaseLocator(folderWorkspaceService.Object, buildConfigProvider, fileSystem, logger);
         }
 
-        private IBuildConfigProvider CreateConfigProvider(string activeConfiguration)
+        private static IBuildConfigProvider CreateConfigProvider(string activeConfiguration)
         {
             var provider = new Mock<IBuildConfigProvider>();
             provider.Setup(x => x.GetActiveConfig(It.IsAny<string>())).Returns(activeConfiguration);
             return provider.Object;
         }
     }
+ 
+    internal static class CompilationDatabaseLocationTestsExtensions
```

Comment:
Extension methods specific to this class.
It only has one method currently, but the new tests for the property evaluator service add more.

Originally, the IFileSystem extensions were in this class, then I realised they were general enough to be worth moving to the test infrastructure  project.